core/domain/subtopic_page_domain.py

```diff
@@ -50,10 +50,10 @@ class SubtopicPageChange(change_domain.BaseChange):
 
     # The allowed list of subtopic page properties which can be used in
     # update_subtopic_page_property command.
-    SUBTOPIC_PAGE_PROPERTIES = (
+    SUBTOPIC_PAGE_PROPERTIES = [
```

Comment:
Why do you change this? If it is because some type requires `List` I think you can change that requirement to `Sequence` or maybe we can enforce `Typle` everywhere as these lists are final and not updated anywhere (I think).

Reply:
We cannot enforce `Tuple` here, because for every argument it would require the number of `str` parameters, wherever it is used, which is different at all places, hence not possible. Using `Sequence`, also throws error suggesting that `List[str]` should be used. So, I think it is the best and strictest way to take care of the situation.